src/nvim/window.c

```diff
@@ -737,6 +737,44 @@ void win_config_float(win_T *wp, FloatConfig fconfig)
     redraw_later(wp, NOT_VALID);
   }
 
+  // compute initial position
+  if (wp->w_float_config.relative == kFloatRelativeWindow) {
+    int row = 0;
+    int col = 0;
+    Error dummy = ERROR_INIT;
+    win_T *win = wp;
+    while (win) {
```

Comment:
is the loop here really needed? Can't we just use `wp->w_winrow` of the parent?

Reply:
Yeah that works, I didn't realize. I've made the change in the latest commit.